### PR TITLE
Update to JWTKit v5

### PR DESCRIPTION
**Pull request by ptoffy**
Update to JWTKit v5 beta 1

### CODE REVIEW

**Comment by 0xTim** [comment]
@ptoffy want to convert this to a draft PR?

File: Package.swift
Comment (0xTim) [inline Package.swift:1]: Need to enable strict concurrency checking to this as well, that will reveal a few more issues

File: Package.swift [hunk 1]
```diff
@@ -1,29 +1,37 @@
...
+        .tvOS(.v16),
+        .watchOS(.v9),
     ],
     products: [
         .library(name: "JWT", targets: ["JWT"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/vapor/jwt-kit.git", from: "4.0.0"),
+        .package(url: "https://github.com/vapor/jwt-kit.git", from: "5.0.0-beta.1"),
         .package(url: "https://github.com/vapor/vapor.git", from: "4.50.0"),
```
Comment (0xTim) [inline Package.swift:17]: Lets bump this to the latest

**Review by 0xTim — APPROVED** [review]

File: Package.swift (hunk 1 above)
Comment (0xTim) [reply Package.swift:17]: Blah

Comment (0xTim) [reply Package.swift:17]: Adding nothing to make GH work

**Review by 0xTim — CHANGES_REQUESTED** [review]

**Comment by 0xTim** [comment]
@ptoffy something got broke

**Comment by ptoffy** [comment]
> @ptoffy something got broke

Yeah that was me confidently editing from the web interface 🤦‍♂️

**Review by 0xTim — APPROVED** [review]

**Pull request merged**